packages/app/src/messages/reducers/messages.interface.ts

```diff
@@ -87,11 +87,20 @@ export type AddNewMessageAction = PayloadAction<
   MessagesEvent.AddNewMessage
 >
 
-export type ToggleThreadReadStatusAction = PayloadAction<
-  string[],
-  MessagesEvent.ToggleThreadReadStatus
+export type ToggleThreadReadStatusPendingAction = PayloadAction<
```

Comment:
Small thing, but shouldn't it be `ToggleThreadsReadStatusPendingAction`? (`Thread`->`Threads`)

Reply:
https://github.com/mudita/mudita-center/pull/743/commits/913da377623a7dede0fa02fac50762d2a117835a